readthedocs/oauth/services/github.py

```diff
@@ -325,7 +325,7 @@ def setup_webhook(self, project, integration=None):
                 recv_data = resp.json()
                 integration.provider_data = recv_data
                 integration.save()
-                log.info('GitHub webhook creation successful for project.')
```

Comment:
I do feel like I'm often trying to debug these things in production, and not having good logs will make that harder. I believe we aren't currently storing debug logs anywhere in prod, so for things that happen rarely and are important, I'm not sure I agree with downgrading them to `debug`.

Reply:
With these ones, in particular, I downgraded the logs for the successful case. However, the ones that failed are still logging in info, I think.